stdlib/public/core/StringGutsRangeReplaceable.swift

```diff
@@ -225,6 +225,33 @@ extension _StringGuts {
     return _foreignConvertedToSmall()
   }
 
+  private mutating func _shouldBeSmall(afterAppendingCount otherCount: Int) -> Bool {
```

Comment:
This shouldn't need to be mutating unless I'm missing something

Reply:
`isUniqueNative` is a `mutating get`.